src/pages/messages/components/NviDisputePage.tsx

```diff
@@ -0,0 +1,3 @@
+export const NviDisputePage = () => {
+  return <p>empty</p>;
+};
```

Comment:
tenker det er lige greit å. bare ikke returnere noe som helst her
```suggestion
export const NviDisputePage = () => {
  return;
};
```

Reply:
Den vil bare merges sammen med de andre uansett